src/test/resources/application.properties

```diff
@@ -24,4 +24,16 @@ NUREGO_BATCH_MAX_MAP_SIZE=1
 NUREGO_BATCH_INTERVAL_SECONDS=1
 METER_BASE_DOMAIN=not-used
 
```

Comment:
All of these credentials should not be in this file. This is a public github. Lets take it offline as to where to put them.